index.html

```diff
@@ -29,14 +36,18 @@ <h1 class="primary-text">Chawki Mbarki</h1>
           and have a project you need coded, don’t hesitate to contact me.</p>
         <button type="button" class="primary-button">Start Collaboration</button>
       </div>
-      <img src="Assets/Shapes/2.svg" alt="geometry-shape">
+      <div class="shape flex-row">
+        <img src="Assets/Images/Headline-image.svg" class="headline-image" alt="geometry-shapes">
+        <img src="Assets/Images/Headline-image2.svg" class="headline-image2" alt="geometry-shapes">
+      </div>
     </section>
 
-    <section class="works">
-      <h1 class="primary-text">Projects</h1>
-      <ul class="projects">
-
-        <li class="project flex-column">
+    <ul class="works max-width">
```

Comment:
- [ ] Please I highly suggest implementing the [FIGMA TEMPLATE](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=23-9&t=Hq64bUbN7WFDPuHZ-0). 😃.

_**REQUIRED FIGMA TEMPLATE**_

![Screenshot (207)](https://user-images.githubusercontent.com/78104544/230527917-ae966186-aac1-4dc8-9109-8432cbe4c993.png)

_**YOUR DESIGN**_

- [ ] Please I highly recommend implementing the project content which involves the project title with an `h2`-`h6` tag, a list with specific technologies and a `button` with hover effects. This design should display when a user hovers over the specific project image per the Figma design. 😄 

![Screenshot (208)](https://user-images.githubusercontent.com/78104544/230528050-b249731d-f170-4ece-898b-5c2ed0e3c6c2.png)